ansible/configs/kni-osp/files/repos_template.j2

```diff
@@ -0,0 +1,44 @@
+[rhel-8-server-rpms]
+name=Red Hat Enterprise Linux 8
+baseurl={{own_repo_path}}/rhel-8-for-x86_64-baseos-rpms/
```

Comment:
I'm not clear on where you are getting your repo path from. Is this meant to be something that is completely contained inside the tenant? Are you expecting to get packages from outside? For RHEL 8, we prefer to use the satellite method...I don't think we have any RHEL 8 content reposync'd down, but we can discuss further.

Reply:
just a leftover from before the content views were created on labsat, going away now